crates/ruff/src/rules/pylint/rules/nested_min_max.rs

```diff
@@ -77,7 +77,18 @@ fn collect_nested_args(context: &Context, min_max: MinMax, args: &[Expr]) -> Vec
                     continue;
                 }
             }
-            new_args.push(arg.clone());
+            let new_arg = if arg.is_generator_exp_expr() {
```

Comment:
I don't think this is quite sufficient, since `max(1, max(a))` will still be flagged, where `a` could be a generator. In fact, I think `a` _has_ to be an iterable, so `max(1, a)` will fail regardless?

Reply:
Indeed, a single `min`/`max` argument has to be iterable according to typeshed.